packages/contentstack-export/src/export/modules-js/marketplace-apps.js

```diff
@@ -43,7 +45,7 @@ module.exports = class ExportMarketplaceApps {
       return Promise.resolve();
     }
 
-    this.developerHubBaseUrl = this.config.developerHubBaseUrl || (await getDeveloperHubUrl(this.config));
+    this.developerHubBaseUrl = this.config.developerHubBaseUrl || (command.developerHubUrl);
```

Comment:
@harshithad0703 @aman19K do we need to update in the old js

Reply:
@aman19K is this backward compatible, for the existing users >?

Reply:
@shafeeqd959 don't think we need to made changes in js code.

Reply:
removed js changes